build/deploy/mkimage.sh

```diff
@@ -13,19 +13,16 @@ set -ex
 cd -P "$(dirname $0)"
 DIR=$(pwd -P)
 
-rm -rf resources ui cockroach
-mkdir -p build
+rm -rf build && mkdir -p build
 docker run -v "$(pwd)/build":/build "cockroachdb/cockroach-dev" shell "cd /cockroach && \
   rm -rf /build/*
   make clean build testbuild >/dev/null 2>&1 && \
   find . -name '*.test' -type f -printf "\"/build/%h \"" | xargs mkdir -p && \
   find . -name '*.test' -type f -exec mv {} "/build/{}" \; && \
-  cp -r resources /build/resources && \
-  cp -r ui /build/ui && \
+  cp -r {ui,resources} /build/ && \
```

Comment:
it's exporting the relevant build artifacts from `/cockroach/` to `/build/`. That's mostly the `cockroach` binary and the `make testbuild` artifacts. `./resources` and `./ui` are also needed for a functional `cockroach` binary, so they're in there as well (I think we should move `ui` into `resources`, by the way. Or get rid of both of them by generating test certs on the fly and embedding the stuff in `ui` directly in our main executable).